install-tbtc.sh

```diff
@@ -14,30 +14,33 @@ printf "${LOG_START}Starting tBTC deployment...${LOG_END}"
 cd "$WORKDIR/relay-genesis"
 npm install
 
-cd "$WORKDIR/tbtc/solidity/migrations"
-
-# Always deploy TestnetRelay instead of the defaull MockRelay.
-jq --arg forceRelay TestnetRelay '. + {forceRelay: $forceRelay}' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
-
-bitcoinTest=$(node "$WORKDIR/relay-genesis/relay-genesis.js")
-BITCOIN_TEST=$(echo "$bitcoinTest" | tail -1)
-
-jq --arg bitcoinTest ${BITCOIN_TEST} '.init.bitcoinTest = $bitcoinTest' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
-jq '.init.bitcoinTest |= fromjson' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
+# If you want to use BTC testnet with real relay maintaner, uncomment the block
+# below and comment the invocation of mock-difficulty.js script.
+#
+# cd "$WORKDIR/tbtc/solidity/migrations"
+# jq --arg forceRelay TestnetRelay '. + {forceRelay: $forceRelay}' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
+#
+# bitcoinTest=$(node "$WORKDIR/relay-genesis/relay-genesis.js")
+# BITCOIN_TEST=$(echo "$bitcoinTest" | tail -1)
+#
+# jq --arg bitcoinTest ${BITCOIN_TEST} '.init.bitcoinTest = $bitcoinTest' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
+# jq '.init.bitcoinTest |= fromjson' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
```

Comment:
What do you think about making it an environment variable or some kind of an option provided to the script?

Reply:
We can but I wonder about just removing this one. To use BTC testnet it's not enough to uncomment here, there are some other places to change. Normally, I don't think we want to use testnet but I left this commented just in case.

Reply:
I can think of a scenario where we want to reuse some of this code to run semi-automatic (with manual BTC transaction) against BTC testnet. I think the approach presented here is the best one for e.g. nightly builds and automatic tests but I am worried we removed difficulty from the picture and it's a key component of the whole system.

Reply:
Ok I get your point. I will add an option here.